code/game/objects/effects/decals/turfdecal/tilecoloring.dm

```diff
@@ -1,6 +1,8 @@
 /obj/effect/turf_decal/tile
 	name = "tile decal"
 	icon_state = "corner_white_full"
+	layer = TURF_PLATING_DECAL_LAYER
```

Comment:
Layer isn't passed to the decal component. You'll have to add in a `, layer` to line 42 of decal.dm for this to have any sort of ingame effect.

Reply:
This wasn't intended to have ingame effect, only correcting the visuals in DM.